duco_api.py

```diff
@@ -4,108 +4,104 @@
 # Distributed under MIT license
 # © Duino-Coin Community 2021
 ##########################################
-from urllib.request import urlopen
 import socket
 from requests import get
-from json import loads
-from threading import Timer
-
-socket.setdefaulttimeout(10)
-ducofiat = .003
 
+API_URL = "https://raw.githubusercontent.com/revoxhere/duco-statistics/master/api.json"
+SERVER_URL = "https://raw.githubusercontent.com/revoxhere/duino-coin/gh-pages/serverip.txt"
 
-def decode_soc(rec):
-    response = rec.decode("utf8")
-    response = response.split(",")
-    return response
+socket.setdefaulttimeout(10)
 
 
-def decode_soc_no_utf(rec):
+def decode_response(rec):
+    """
+    Decodes a response from the server
+    """
     response = rec.decode()
     response = response.split(",")
     return response
 
 
-def GetDucoPrice():
```

Comment:
Why remove the get duco price?

Reply:
Oh, I'm dumb, I just saw it not being used anywhere and removed it, not realizing that I'm editing a module. I will add it back, give me a second

Reply:
I've made a commit, please check.